### ISSUE TITLE
SoftMax derivative with crossentropy  i Rapport

### ISSUE THREAD

**Issue by XxRedPanda** [GoodVibeSquad/first_itteration#101]
Add softmax derivative with crossentropy after the MSE section in the report. Explain why we use softmax with crossentropy in a shorter section.
